.github/workflows/_tauri.yml

```diff
@@ -36,12 +36,12 @@ jobs:
             syms-artifact: rust/gui-client/firezone-linux-gui-client-amd64.dwp
             pkg-artifact: rust/gui-client/firezone-linux-gui-client_amd64.deb
           - runs-on: windows-2019
-            binary-dest-path: firezone-windows-client
+            binary-dest-path: firezone-client-gui-windows
             rename-script: ../../scripts/build/tauri-rename-windows.sh
             upload-script: ../../scripts/build/tauri-upload-windows.sh
-            exe-artifact: rust/gui-client/firezone-windows-client-x64.exe
-            syms-artifact: rust/gui-client/firezone-windows-client-x64.pdb
-            pkg-artifact: rust/gui-client/firezone-windows-client-x64.msi
+            exe-artifact: rust/gui-client/firezone-client-gui-windows-x86_64.exe
```

Comment:
If we're changing these to `x86_64` we should probably do the gateway too right?

Reply:
Yeah. It feels a bit pointless because I could definitely map "x86_64" to "x64" in the Rust code. I'm just doing it so that the update code can use Rust's `std::env::consts::ARCH` directly. But maybe it's not worth it?

Reply:
Let's change it then -- we'll soon have permalinks anyway

Reply:
Well, hmmm that'll require changing a bit of CI configuration. I'll see if I can do it in my commits.

Reply:
Updated